app/jobs/submit_iplc_listener_job.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+##
+# Background job to send requests to IPLC.
+# If Reshare has the item and we succesfully request it, let the user know it is on the way.
+# If Reshare does not have it or we cannot succesfully request it, trigger the normal Symphony request workflow.
+class SubmitIplcListenerJob < ApplicationJob
+  discard_on ActiveRecord::RecordNotFound do |job, _error|
+    Honeybadger.notify(
+      "Attempted to call Reshare for Request with ID #{job.request_id}, but no such Request was found."
+    )
+  end
+
+  def perform(request_id, instance_uuid)
+    request = Request.find(request_id)
+
+    Sidekiq.logger.info("Started SubmitIplcRequestJob for request #{request_id}")
+
+    begin
+      make_iplc_request(request, instance_uuid)
+    rescue StandardError => e
```

Comment:
I feel like this might be too generic.  Would it make sense for line 33 to raise a specific error and rescue it here?

Reply:
@cbeer may be able to speak to this more confidently, but my understanding is that we rescue from `StandardError` because if anything goes wrong we want to fall back to sending the request to Symphony.

FWIW the existing `SubmitBorrowDirectRequestJob#perform` method also wraps the request call in a rescue block that rescues `StandardError` -- so it's not a change from the current implementation: https://github.com/sul-dlss/sul-requests/blob/master/app/jobs/submit_borrow_direct_request_job.rb#L16

Reply:
It seems to me this isn't exactly clear.  If that's indeed @cbeer's intent, it would be helpful to have some documentation on make_iplc_request that says what errors it expects to throw and when.

Reply:
> if anything goes wrong we want to fall back to sending the request to Symphony.

That's my understanding too. We'd prefer to get the book from some other library, but not getting the material in some way isn't an option. 

My suggestion would be... we merge this as-is, see how reshare actually breaks, and revisit it when we have time. If you have a comment in mind ("yes, we actually mean it when we're rescuing everything"?), feel free to add it.

Reply:
Given that we're up against a deadline, I'm going to merge this as-is and we can revisit if needed.